### DISCUSSION TITLE
I cant install

### DISCUSSION THREAD

**Discussion by jagra26** [bytebeamio/rumqtt#310] [Q&A]
i follow the instructions of the crates, but i give a bunch of erros:

![image](https://user-images.githubusercontent.com/31116911/134684492-68404bbd-32c3-46a7-91b9-efec45d812c1.png)

![image](https://user-images.githubusercontent.com/31116911/134684566-03a479ea-4321-4c5a-b534-1bb8de45dc9d.png)

![image](https://user-images.githubusercontent.com/31116911/134684607-fc0f56a3-653d-4b79-bb33-9267b336bc06.png)

How do i  proceed?

**Comment by gautambt** [comment]
Hello, in case you are still looking for an answer, this was a problem in 0.8.0 version. Please upgrade to the latest version and let us know if you are facing any issues.
